keps/prod-readiness/sig-scheduling/3022.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 3022
+alpha:
+  approver: "@wojtek-t"
```

Comment:
@wojtek-t
Hello. 
I assigned it to you. Please let me know if I should assign someone other than you 🙇

Reply:
Thanks.

> next time please assign PR to me too, notifications aren't useful for me at that scale

Sure. 🙇